docs/asciidoc/modules/ROOT/pages/ml/bedrock.adoc

```diff
@@ -0,0 +1,225 @@
+"token": "▁Review",
+"raw_logprob": -7.870129585266113,
+"logprob": -7.870129585266113
+}
+}
+,
+{ ....
+| [
+{
+"data": {
+"text": "
+These cabinets are very old and outdated. They are made of wood and have a simple, classic design. The cabinets are in good condition, but they could use some updating.",
+"tokens": [
+{ ...
+|===
+
+
+We can also change the modelId, for example by using the `"ai21.j2-mid-v1"` one, which return the same response values.
+[source,cypher]
+----
+CALL apoc.ml.bedrock.jurassic(<BODY>, {modelId: "ai21.j2-mid-v1"})
+----
+
+
+
+.apoc.ml.bedrock.anthropic.claude
+[source,cypher]
+----
+CALL apoc.ml.bedrock.anthropic.claude({
+                prompt: "\n\nHuman: Hello world\n\nAssistant:",
+            max_tokens_to_sample:  300,
+            temperature: 0.5,
+            top_k: 250,
+            top_p: 1,
+            stop_sequences: ["\\n\\nHuman:"],
+            anthropic_version: "bedrock-2023-05-31"
+})
+----
+
+
+.Results
+[opts="header",cols="2"]
+|===
+|  completion| 	stopReason
+| " Hello!"	| "stop_sequence"
+|===
+
+
+.apoc.ml.bedrock.titan.embed
+[source,cypher]
+----
+CALL apoc.ml.bedrock.titan.embed({inputText: "Hello World"})
+----
+
+.Results
+[opts="header",cols="2"]
+|===
+|  inputTextTokenCount| 	embedding
+| 2	| 	[0.32421875, 0.35546875, 0.625, 0.20019531, 1.328125, 0.6171875, 0.11425781, -0.00074005127, .... 
+|===
+
+
+.apoc.ml.bedrock.stability
+[source,cypher]
+----
+CALL apoc.ml.bedrock.stability({
+text_prompts: [{text: "picture of a bird", weight: 1.0}],
+cfg_scale: 5,
+seed: 123,
+steps: 70,
+style_preset: "photographic"
+})
+----
+
+.Results
+[opts="header"]
+|===
+|  base64Image
+| "iVBORw0KGgoAAAANSUhEUgAAAgAAAAIACAIAAAB7GkOtAAABjmVYSWZNTQAqAAAACAAGAQAABAAAAAEAAAIAAQEABAAA...."
+|===
+
+
+
+=== List of models
+
+[source,cypher]
+----
+CALL apoc.ml.bedrock.list()
+----
+
+.Results
+[opts="header"]
+|===
+| modelId                           | modelArn                                                             |modelName                   |providerName  |responseStreamingSupported|customizationsSupported|inferenceTypesSupported|inputModalities  |outputModalities
+| "amazon.titan-tg1-large"          |"arn:aws:bedrock:us-east-1::foundation-model/amazon.titan-tg1-large"  |"Titan Text Large"          |"Amazon"      |true                      |["FINE_TUNING"]        |["ON_DEMAND"]          |["TEXT"]         |["TEXT"]        
+| "amazon.titan-e1t-medium"         |"arn:aws:bedrock:us-east-1::foundation-model/amazon.titan-e1t-medium" |"Titan Text Embeddings"     |"Amazon"      |null                      |[]                     |["ON_DEMAND"]          |["TEXT"]         |["EMBEDDING"]
+| ...       |... |...     |...     |null                      |[]                     |...         |...         |...
+|===
+
+
+== Custom AWS API Call
+
+Via the `apoc.ml.bedrock.custom` we can create a customizable Bedrock API Request, by choosing the HTTP Method, the endpoint, the region and the additional headers.
```

Comment:
we can definitely keep the custom, just need the same base procedures as the other providers

and not tie ourselves in procedure names to model names, if they change their names or discontinue models it's much harder to adjust than changing default models in the config.